travis-build

```diff
@@ -11,26 +11,27 @@ pip install dist/*.tar.gz
 omego version
 omego -h
 
-#Install a new server
+#Install a new server without web
 #Tests rely on a non-zero error code being returned on failure
 if [ $TEST = install ]; then
-  omego install --initdb --dbhost localhost --dbname omero --prestartfile $HOME/config.omero -v --release 5.0.8 --ice 3.5
+  omego install --initdb --dbhost localhost --dbname omero --prestartfile $HOME/config.omero -v --release 5.3.5 --ice 3.6 --no-web
 
   # Should return 0 DB_UPTODATE
-  omego db upgrade -n --dbhost localhost --dbname omero --serverdir OMERO.server-5.0.8-ice35-b60
+  omego db upgrade -n --dbhost localhost --dbname omero --serverdir OMERO.server-5.3.5-ice36-b73
 
   # Check the expected server version was downloaded
-  test $(readlink OMERO.server) = './OMERO.server-5.0.8-ice35-b60'
+  test $(readlink OMERO.server) = './OMERO.server-5.3.5-ice36-b73'
 
   # Check db dump file
   omego db dump --serverdir OMERO.server --dumpfile travis-omero.pgdump
-  pg_restore -e travis-omero.pgdump | grep 'CREATE TABLE dbpatch'
+  
+  pg_restore -l travis-omero.pgdump | grep 'dbpatch_versions_trigger'
 fi
 
-#Test a multistage DB upgrade (4.4 -> 5.3) as part of the server upgrade
+#Test a multistage DB upgrade (5.3 -> 5.4) as part of the server upgrade
```

Comment:
Could you make this 5.2 -> 5.4 instead (https://downloads.openmicroscopy.org/omero/5.2.8/ includes an ice 3.6 server)? The intention is to test an OMERO DB upgrade where there's not a direct upgrade path, e.g. to go from 5.2 to 5.4 you must run two files:
- `sql/psql/OMERO5.3__0/OMERO5.2__0.sql`
- `sql/psql/OMERO5.4__0/OMERO5.3__0.sql`

Reply:
I did not check that 
Thanks

Reply:
and we will be going to 5.5 as part of this release

Reply:
I did a bit of investigation:
* 5.2 server will not start on Xenial travis docker image due to the Java version installed by default.
* An option is to keep the 14.04 image and use the wheel for 14.04 (we will need for https://github.com/openmicroscopy/openmicroscopy/ not available yet)
* then move to 16.04 image when we have 5.5 out so we can do an upgrade from 5.3-> 5.5

What do you think?

Reply:
Assuming no omego work is planned we could keep it as 5.3->5.4, and add an issue as a reminder to change it to 5.5?

Reply:
sounds good. So I will roll back the commits and when we have the wheel in its final location
update the pip installation

Reply:
https://github.com/ome/omego/issues/117

Reply:
Thanks, this looks fine now